### ISSUE TITLE
[VPS-173] (2) The download button on the manage groups page is functional

### ISSUE THREAD

**Issue by JordanBlenn** [UoaWDCC/VPS#173]
**Description:**

**AS A:** Staff member creating group scenarios
**I WANT:** the download button to download group data with the associated scenario
**SO THAT:** I can easily send out emails to the students with their given scenarios to play through a service such as MailChimp or equivalent automated personal email platform.

**AC's:**
1) When the Download CSV button is clicked, it downloads a CSV file of the groups.
2) The info contained is separated as follows.:     email | first name | last name | playable link
3) The playable link is formatted as follows: https://vps-dev.wdcc.co.nz/play/{scenarioId}/{groupId}

**Technical Discussion:**
- Roles should not be required in this info as they are linked to the email address, but double check with @wjin-lee first (or maybe we want to mention what role somebody is when they get the email telling them to complete the scenario?)
- Blocked by #154 (need to have the required info in the database first)
- Blocked by #170 (need to have the required info pulled from db to frontend first)

**Comment by wjin-lee** [comment]
**AC 2 and 3 seem to be outdated.**

For reference, I would have expected something along the lines of:
```csv
email, name, role, group_number, link
abc@gmail.com, abc aba, doctor, 1, https://vps-dev.wdcc.co.nz/play/65e45789a86f985e1c5af28f 
xyz@gmail.com, xyz xyx, nurse, 1, https://vps-dev.wdcc.co.nz/play/65e45789a86f985e1c5af28f 
```
(Note that the link would be the same for everyone due to how we ended up architecting the `/play/:scenarioId` page)

**Comment by JordanBlenn** [comment]
Agreed, let's discuss in retro. 
I tried to specify what was required without fully understanding the technical base for it - my bad. 
VC 2 - was just what I though was necessary information but having the group and role numbers is fine. Only reason I didn't include it is to make it simpler for the lecturers/staff using it - wanted to make it as simple as possible.
For VC 3 - that change to the link format was updated after I made the story and then I was never actually updated about the change,
